packages/components/scroll-area/src/props.ts

```diff
@@ -430,7 +431,8 @@ export const scrollAreaScrollbarProps = {
   props: {
     ...scrollAreaScrollbarVisibleProps.props,
     forceMount: {
-      type: Boolean as PropType<true | undefined>,
+      type: Boolean,
+      default: true,
```

Comment:
no true, true is Type `type: Boolean as PropType<true | undefined>,`

Reply:
`forceMount` true interface you should pay attention that it is a type not a default value. it says it can be true or undefined.

Reply:
default: undefined,

Reply:
Hmmm, but the thumb was't showing by doing that. When I changed it, the thumb appears.

Reply:
> `forceMount` true interface you should pay attention that it is a type not a default value. it says it can be true or undefined.

Ooh, I see, i will revert that change.